src/commands/commit.ts

```diff
@@ -24,11 +28,35 @@ export async function _commit(parsedSettings: ParsedSettings): Promise<void> {
   if (Number.isNaN(newMigrationNumber)) {
     throw new Error("Could not determine next migration number");
   }
-  const newMigrationFilename =
-    String(newMigrationNumber).padStart(6, "0") + ".sql";
-  const currentMigrationPath = getCurrentMigrationPath(parsedSettings);
-  const body = await fsp.readFile(currentMigrationPath, "utf8");
-  const minifiedBody = pgMinify(body);
+
+  // See if we have a message arg
+  const messageIndex =
+    process.argv.findIndex(arg => arg === "--message" || arg === "-m") + 1;
```

Comment:
Normally we use `-1` for doesn't exist, but this would result in `0` which is unexpected

```
> [].findIndex(() => false) + 1
0
```

Better would be to get the index first, then add 1 if it exists, otherwise have it be `null` or similar.

Reply:
True, way less hacky. Fixed!